### PR TITLE
Added supported browser detection.

### PR DESCRIPTION
**Pull request by thealmightyone**
Added login screen error message.

### CODE REVIEW

**Comment by stepek** [comment]
Please move browser support check to other file and include it to index html

File: app/index.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
+import './js/browserDetect';
 import ConfigModel from './js/models/configModel';
 import SchemaCollection from './js/models/schemaCollection';
 import SchemaModel from './js/models/schemaModel';
 import UserModel from './js/models/userModel';
 
 import Router from './js/routers/router';
 
+
```
Comment (stepek) [inline app/index.js]: Please remove this line :)

File: app/js/browserDetect.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  }
+  if (match[1] === 'Chrome') {
+    temp = userAgent.match(/\b(OPR|Edge)\/(\d+)/);
+    if (temp !== null) return temp.slice(1).join(' ').replace('OPR', 'Opera');
+  }
+  match = match[2] ? [match[1], match[2]] : [navigator.appName, navigator.appVersion, '-?'];
+  if ((temp = userAgent.match(/version\/(\d+)/i)) !== null) {
+    match.splice(1, 1, temp[1]);
+  }
+  if (supportedBrowserSet.hasOwnProperty(match[0]) && supportedBrowserSet[match[0]] <= match[1]) {
```
Comment (thayashi) [inline app/js/browserDetect.js]: Shouldn't be `supportedBrowserSet[match[0]] > match[1]`?
Also match[1] is String, it might be better to change Number.

Comment (thealmightyone) [inline app/js/browserDetect.js]: Wow my bad, it wasnt actually working because I thought I had Chrome 50

**Comment by thayashi** [comment]
LGTM

**Pull request merged**